server/index.js

```diff
@@ -0,0 +1,99 @@
+const dotenv = require("dotenv");
+const express = require("express");
+const fs = require("fs");
```

Comment:
You introduce the local test server to version 1 so these dependencies must be added as well. In versions that already have the local test server only "dotenv" must be added as dev dependency.

Reply:
And the information must be added to README.md

Reply:
You can also decide to first create a PR to add the test server to version 1. And continue from there to reduce the amount of conflicts.

Reply:
I've cherry-picked the addition of the local image server to add the test server to v1.0.

Reply:
Only the `dotenv` dev requirement is still missing.